### ISSUE TITLE
Homekit displays "no response" after a time period until homebridge is restarted.

### ISSUE THREAD

**Issue by apressly** [nfarina/homebridge#843]
I am having an issue with homebridge where I see "no response" on the homekit app on ios 10 after a couple minutes and need some help. 

1.I have home bridge running on either screen or terminal on a mac mini OS sierra that is always on.
2. My apple tv is set to never sleep
3. I set up a ping for every 30 min to ping my apple tv and verify its not sleeping
4. I see the home bridge is running fine on the mac but after a set time, if I look at my iPhones or apple watch, home kit says "no response" or "updating" and then "no response".
5. I have an iphone 6 running ios 10

Quiting the home bridge process and re-running it fixes the issue for a time but I am not always near my computer nor want to do that.

my set up is the latest home bridge with Node: v4.6.0 and NPM: 3.10.9 with the following plugins: running the newer nam i get a different error
a. Harmony hub
b. Nest
c. Chamberlain MyQ garage door.

I am not sure if there is any fix but I was wondering if there is a way to run this such that it can be automatically quit and re run every so often if there is no fix.

**Comment by lagunacomputer** [comment]
mae sure HomeBridge is not crashing exactly at the time of event.
sudo systemctl status -l homebridge

for instance, mine was loading, but then crashing upon request for update because of a bad plugin.

**Comment by apressly** [comment]
Were you able to fix it? Which plugin was bad from yours?

**Comment by apressly** [comment]
sudo systemctl status -l homebridge does not work- no command found

**Comment by lagunacomputer** [comment]
do you start homekit on the raspi via systemctl?

see:
https://github.com/nfarina/homebridge/wiki/Running-HomeBridge-on-a-Raspberry-Pi
"Running Homebridge on Bootup (systemd)"

**Comment by apressly** [comment]
its not a raspy.. its a mac mini using mac os sierra

**Comment by lagunacomputer** [comment]
do you start homekit on the mac mini via systemctl?

see:
https://github.com/nfarina/homebridge/wiki/Running-HomeBridge-on-a-Raspberry-Pi
"Running Homebridge on Bootup (systemd)"

**Comment by apressly** [comment]
i started homebridge via launchctl https://github.com/nfarina/homebridge/wiki/Install-Homebridge-on-OSX

**Comment by apressly** [comment]
Just an update  on this topic, I am doing some troubleshooting to see if it is a problem with one of the packages. Last night I went ahead and uninstalled Nest and MyQ garage door and only left the harmony hub. It does say not responding but once  I tap it, it shows up again so  there is a problem still but it's better than before I don't need to restart the bridge. Tonight I will uninstall harmony and only install nest and see how long it lasts.

**Comment by Kolbi** [comment]
Seems to be the same issue: https://github.com/nfarina/homebridge/issues/849 ? #849

**Comment by apressly** [comment]
Update I got mine working... there were 2 issues:
1. My Q -> use Liftmaster 2 plugin
2. Homekit on spouses phone, -delete "default home" 

 1) I got rid of myQ and using Liftmaster 2 plugin.. now with harmony, liftmaster 2, wink and nest.. all is working.

. Also i had to use    this for wink:
"client_id": "quirky_wink_android_app",
"client_secret": "e749124ad386a5a35c0ab554a4f2c045",

2) The iphone that has the homekit sign in which is attached to the apple tv 4 is the main phone, you need to invite your spouse or kids on their phones and give them all access. They need to delete the default "Home" in their app and use the family home that was set up on the original phone

I am closing this as mine is working

**Comment by apressly** [comment]
Just to be clear all the steps i took:

Quit homebridge from console
Uninstalled all plugins from console
Delete bridge from Home app on iphone
On Mac Delete persist and accessory files

Next install one plugin at a time to make sure the plugins are good. 

Every time I installed a plugin I  Quit home bridge, updated the config file delete the bridge in home app and delete persist and accessory files then restarted a fresh home bridge in console and re-added bridge in on home app. I would wait a day or at least a couple hours to make sure its still responding, turned off wifi, turned on wifi to make sure I could access on network and off network.

I installed harmony hub, let it run , Then installed nest , then liftmaster2, then wink
